index.html

```diff
@@ -64,98 +92,122 @@ <h2 class="Title-post-1">Multi-Post Stories</h2>
                                     <button class="see-project-1" href="">See project</button>
                             </div>
                         </div>
-                        <div class="card-2 card">
-                            
-                            <h2 class="Title-post">Profesional Art Printing Data</h2>
-                            <p class="Supporting-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-                            <div class="tags">
-                                <div>
-                                    <ul class="tag-link">
-                                        <li class="tag "><a href="https://www.html.org">html</a></li>
-                                        <li class="tag "><a href="https://getbootstrap.com/">bootstrap</a></li>
-                                        <li class="tag "><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
-                                    </ul>
+                        <div class="card-sec">
+                                <div class="card-2 card card-sec-1">
+                                    
+                                    
+                                        <h2 class="Title-post mob">Profesional Art Printing Data</h2>
+                                        <h2 class="Title-post desk">Data Dashboard Healthcare</h2>
+                                        <p class="Supporting-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                                    
+                                    <div class="tags deskoff">
+                                        <div>
+                                            <ul class="tag-link ">
+                                                <li class="tag "><a href="https://www.html.org">html</a></li>
+                                                <li class="tag "><a href="https://getbootstrap.com/">bootstrap</a></li>
+                                                <li class="tag "><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+                                            </ul>
+                                        </div>
+                                        <span class="see-project see-project-vis "><a href="">See project</a></span>
```

Comment:
It seems that this button has misaligned while adding some query. Try to make a design as close to FIGMA design as possible.

| Your Design  | FIGMA Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/213812896-928b2fc6-f088-47af-9e82-ff247b1cdf56.png)  | ![image](https://user-images.githubusercontent.com/61361037/213812956-707972f3-2e79-4031-82bb-433a9a9be714.png)  |